### ISSUE TITLE
Update fade behaviour

### ISSUE THREAD

**Issue by Richard-L** [rr-/Tomb1Main#379]
Granted this is very meticulous, and perhaps a bit too much of personal preference, but on the console versions the black fades do not animate over the ring menu animation. So as the game loads it would fade up the main menu backdrop, and then once the fade is complete animate out the ring with the music starting. Currently the animation and fade up happens at the same time.

Similarly the other way round, when a level is loaded or started, the ring menu animates in, and _then_ the fade takes place.

I understand if you'd rather have it the way you've done it as of v2.5. I just wish we could maintain console similarity for as long as possible.

Generally speaking the newly added fades are very nice.

Another thing I noticed is that the gym level has no fade out upon completion and Lara's "better take off these wet clothes" line. The game should freeze like it were a level end with the stat screen appearing, just that the stats are skipped altogether.

**Comment by rr-** [comment]
@Richard-L can you split adding a fade effect to the gym level end into a separate issue?

**Comment by Richard-L** [comment]
I looked into this a bit more, and the fade does actually animate over the ring menu whilst you're in a level already. TR1M has pretty much nailed this behaviour now to the point where it's indistinguishable from console (second half of [first video](https://user-images.githubusercontent.com/4934209/150340539-d891aec9-fdd5-4be2-a43f-cd1d9f9787d7.mp4) in #333).

The main menu ring menu is a bit of a particular case, as described above, and the main concern of this issue.

Upon game launch:

https://user-images.githubusercontent.com/4934209/152049416-9f09451b-061c-4e89-9b5d-5bc592eefffc.mp4

^where obviously we only need the fade-out + ring-animate-out part at the end.

Upon level start:

https://user-images.githubusercontent.com/4934209/152049495-4748d850-9e49-48e9-9fc3-7a22e7b3d251.mp4

I'm very grateful you have an ear for these nuances.

**Comment by rr-** [comment]
@walkawayy do you happen to remember how we can test whether we're in the double-FPS inventory or single-FPS game menu?

**Comment by walkawayy** [comment]
I honestly am not sure what you mean.

**Comment by rr-** [comment]
Inventory runs its logic at 60 FPS AFAIR.

**Comment by Richard-L** [comment]
Whilst I don't know myself, I would believe this to be true, since Saturn has 60fps inventory natively (a yet to be posted feature request :innocent: )
